bootstrap/integration/hooks_integration_test.go

```diff
@@ -488,3 +490,85 @@ func TestPreExitHooksFireAfterCancel(t *testing.T) {
 
 	tester.CheckMocks(t)
 }
+
+func TestPolyglotScriptHooksCanBeRun(t *testing.T) {
+	if runtime.GOOS == "windows" {
+		t.Skip("script hooks aren't supported on windows")
+	}
+
+	path, err := exec.LookPath("ruby")
+	if err != nil {
+		t.Fatal("error finding path to ruby executable: %w", err)
```

Comment:
IIRC `%w` is only for `fmt.Errorf`, and also `t.Fatal` doesn't do formatting verbs at all.

```suggestion
		t.Fatalf("error finding path to ruby executable: %v", err)
```

This might be caught with `go vet`?

Reply:
when i changed `%w` for `%v` it came up with the lil squiggly yellow line, i'm guessing that it doesn't see the %w verb as relevant to calls to `t.Fatal`? fair enough i suppose.

fixed now, btw